File: SampleSetup/Protocol/Model/ApiAddress.cs
using AbstractModelWithId = SampleProject.Protocol.Model.AbstractModelWithId;
using ApiCountry = SampleProject.Protocol.Model.ApiCountry;
using ApiCity = SampleProject.Protocol.Model.ApiCity;
using System;
using System.Collections.Generic;
using System.Text;

namespace SampleProject.Protocol.Model
{
    public class ApiAddress : AbstractModelWithId
    {
        public string CountryId { get; set; }

        public ApiCountry Country { get; set; }

        public string CityId { get; set; }

        public ApiCity City { get; set; }

        public string PostalCode { get; set; }

        public string AddressLine1 { get; set; }

        public string AddressLine2 { get; set; }

        public double? GeoLat { get; set; }

        public double? GeoLon { get; set; }

        public string Description { get; set; }
    }
}

File: SampleSetup/Output/PHP_8/Model/ApiAddress.php
<?php
namespace SampleProject\Protocol\Model;

class ApiAddress extends \SampleProject\Protocol\Model\AbstractModelWithId {
    public string $CountryId;

    public \SampleProject\Protocol\Model\ApiCountry $Country;

    public string $CityId;

    public \SampleProject\Protocol\Model\ApiCity $City;

    public string $PostalCode;

    public string $AddressLine1;

    public string $AddressLine2;

    public ?float $GeoLat;

    public ?float $GeoLon;

    public string $Description;
}
